cypress/e2e/3-default-owner.cy.ts

```diff
@@ -26,90 +26,92 @@ const lastName = randLastName();
 describe('Default owner', () => {
 	// todo test should redirect to setup if have not skipped
 
-	beforeEach(() => {
+	before(() => {
 		cy.resetAll();
 	});
+	beforeEach(() => {
+		cy.skipSetup();
```

Comment:
shouldn't this also be in `before` ? unless you are resetting the db, you shouldn't have to worry about skipping the setup.

Reply:
ya this makes sense.. only need to skip once... that's why it worked as one complete test